scalanet/src/io/iohk/scalanet/peergroup/dynamictls/DynamicTLSPeerGroupInternals.scala

```diff
@@ -228,9 +256,9 @@ private[dynamictls] object DynamicTLSPeerGroupInternals {
       serverQueue: CloseableQueue[ServerEvent[PeerInfo, M]],
```

Comment:
Off topic: it would be so good to clean up the `ServerChannelBuilder` which is used as a side effecting constructor in `DynamicTLSPeerGroup.serverBootstrap`. Every time I look at this code I'm lost about what `ServerChannelImpl` and `ClientChannelImpl` are; only by tracking where they are created/called from can I put myself in the frame of figuring out which event is raised where and what the Netty channels represent.

Probably because in my mind the "server channel" is where incoming connections are accepted, whereas it just means "incoming client channel" vs "outgoing client channel". But at least if it was a static builder function it would indicate more clearly how its used. Maybe `ServerChannelImpl.build` or `apply`.

Reply:
Seems like good idea, although I would to this after my next pr which touches a little bit this area. (I will create task there or some kinda TODO)

Reply:
Thanks :+1: